From 74dcf17e46bfed24be538145bd6230cac2583acc Mon Sep 17 00:00:00 2001
From: Igor Grinberg <grinberg@compulab.co.il>
Date: Mon, 1 Jul 2013 14:58:47 +0300
Subject: video: mxc: lcd: adjust SCF0403 LCD timings

Adjust DataImage SCF0403 LCD timings for better image position on
various variants of the panel.

Signed-off-by: Igor Grinberg <grinberg@compulab.co.il>
---
 drivers/video/mxc/mxc_lcdif.c |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/drivers/video/mxc/mxc_lcdif.c b/drivers/video/mxc/mxc_lcdif.c
index 8181098..b18bbf8 100644
--- a/drivers/video/mxc/mxc_lcdif.c
+++ b/drivers/video/mxc/mxc_lcdif.c
@@ -40,17 +40,17 @@ static struct fb_videomode lcdif_modedb[] = {
 	FB_VMODE_NONINTERLACED,
 	0,},
 	{
-	/* 480x800 @ 60 Hz , pixel clk @ 29MHz */
+	/* 480x800 @ 60 Hz , pixel clk @ 25MHz */
 		.name		= "SCF04-WVGA",
 		.xres		= 480,
 		.yres		= 800,
-		.pixclock	= 34722,
-		.left_margin	= 12,
-		.right_margin	= 16,
-		.upper_margin	= 30,
-		.lower_margin	= 40,
-		.hsync_len	= 26,
-		.vsync_len	= 30,
+		.pixclock	= 40510,
+		.left_margin	= 3,
+		.right_margin	= 18,
+		.upper_margin	= 10,
+		.lower_margin	= 10,
+		.hsync_len	= 1,
+		.vsync_len	= 10,
 		.vmode		= FB_VMODE_NONINTERLACED,
 	},
 };
-- 
1.7.9.5

